basics/basic-types.md

```diff
@@ -0,0 +1,124 @@
+# Grundläggande typer
+
+D erbjuder ett antal grundläggande typer som alltid har samma
+storlek **oavsett** vilken plattform. Det enda undantaget
+är typen `real` som erbjuder den högsta möjliga precision för
+flyttal. Det finns inga skillnader för heltals storlekarna oavsett
+om programmet är kompilerat för ett 32-bitar eller ett 64-bitars system.
+
+| typ                           | storlek
+|-------------------------------|------------
+|`byte`                         | 8-bit
+|`bool`, `ubyte`, `char`        | 8-bit
+|`short`, `ushort`, `wchar`     | 16-bit
+|`int`, `uint`, `dchar`         | 32-bit
+|`long`, `ulong`                | 32-bit
+
+#### Flyttalstyper:
+
+| typ     | storlek
+|---------|--------------------------------------------------
+|`float`  | 32-bit
+|`double` | 64-bit
+|`real`   | >= 64-bit (generellt 64-bit, men 80-bit på Intel x86 32-bit)
+
+Prefixet `u` betecknar en *osignerad* typ. `char` översätter till
```

Comment:
`översätter` sounds weird in this context. It's also inconsistent with the description of the `wchar` and `dchar`. `char` mentions characters, while the other two mention strings.